tools/cli/workflow.go

```diff
@@ -364,3 +370,89 @@ func newActivityCommands() []cli.Command {
 		},
 	}
 }
+
+func newBatchCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:    "describe",
+			Aliases: []string{"desc"},
+			Usage:   "Describe a batch operation job",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  FlagJobIDWithAlias,
+					Usage: "Batch Job ID",
+				},
+			},
+			Action: func(c *cli.Context) {
+				DescribeBatchJob(c)
+			},
+		},
+		{
+			Name:  "stop",
```

Comment:
s/stop/terminate to be consistent with workflows

Reply:
Done